testing/trino-testing/src/main/java/io/trino/testing/AbstractTestDistributedQueries.java

```diff
@@ -597,6 +597,20 @@ public void testInsertUnicode()
         assertUpdate("DROP TABLE " + tableName);
     }
 
+    @Test
+    public void testInsertHighestUnicode()
+    {
+        skipTestUnless(supportsInsert());
+
+        String tableName = "test_insert_unicode_" + randomTableSuffix();
+
+        assertUpdate("CREATE TABLE " + tableName + "(test varchar(50))");
+        assertUpdate("INSERT INTO " + tableName + "(test) VALUES 'Hello', U&'hello\\6d4B\\8Bd5\\+10FFFFworld\\7F16\\7801' ", 2);
```

Comment:
remove other non-ASCII that are not 10fff
keep other asciis as might help with debugging